k4FWCore/components/UniqueIDGenSvc.h

```diff
@@ -35,11 +35,13 @@
  */
 class UniqueIDGenSvc : public extends<Service, IUniqueIDGenSvc> {
 public:
+  using seed_t = uint64_t;
+
```

Comment:
Should this potentially also live in the `IUniqueIDGenSvc` like the `event_num_t` and `run_num_t`?

Reply:
That makes sense.